packages/create-next-app/package.json

```diff
@@ -36,6 +36,7 @@
     "@zeit/ncc": "^0.20.4",
     "async-retry": "1.3.1",
     "chalk": "2.4.2",
+    "cli-prompts-test": "0.2.0",
```

Comment:
We generally don't add packages that are from the immediate contributors to reduce risk. It seems like the package is only one file so it could live inside the Next.js test suite.